unimath-symbols.ltx

```diff
@@ -52,22 +52,22 @@
 \def\INPUT{\input{unicode-math-table.tex}}
 \def\TABLE{%
 \par\noindent
-\begin{longtable}[l]{@{}lccccccccll@{}}
+\begin{longtabu}[l]{@{}lcccccccclX[l]@{}}
   \toprule
   \textsc{usv} & M & X & S & C & A & P & D & E & Macro & Description \\
   \midrule \endhead
   \INPUT\\
   \bottomrule
-\end{longtable}
+\end{longtabu}
 }
 \makeatletter
 \def\USV#1{\footnotesize\scshape\MakeLowercase{u+\@gobble#1}}
 \makeatother
 \def\CMD#1{\footnotesize\cmd#1}
 \def\DESC#1{%
-  \begin{varwidth}[t]{4cm}
-    \raggedright\linespread{0.6}\scriptsize #1%
-  \end{varwidth}
+  \begin{RaggedRight}\begin{spacing}{0.5}
```

Comment:
`\RaggedRight` isn't an environment — it would be better to write either `\begin{spacing}{0.5}\RaggedRight`, or using the `FlushLeft` environment instead.